### PR TITLE
removes root; adds urist-friendly ipc lore

### PR DESCRIPTION
**Pull request by ggermanicus**
except for removing root i have tried to make pretty neutral blurbs. ipcs can now be from anywhere a human can be. this includes terra in the niche case of ipcs that are from pre-terra terra (new earth) or players with better ideas but i can remove that if we aren't feeling it
also fixes a typo in ceti epsilon

### CODE REVIEW

File: maps/nerva/fluff/locations_human.dm [hunk 1]
```diff
@@ -114,13 +114,13 @@
 	Even over a decade after the end of the Galactic Crisis, cleanup efforts are still ongoing in some areas of the planet, as previously \
 	undiscovered alien hives are uncovered. Although separated from their hivemind, individual aliens still are a threat to those trying \
 	to return to their lives that were torn apart by war. The bright point of the rebuilding process is the Ceti Institute of Technology, which \
-	has been completely rebuilt, and is slowly returning to its former glory. br><br>The people of Ceti Epsilon are some of the most, well, average in the TC. Being from a largely built up world with a focus on \
+	has been completely rebuilt, and is slowly returning to its former glory. <br><br>The people of Ceti Epsilon are some of the most, well, average in the TC. Being from a largely built up world with a focus on \
 	the cutting edge, most Episoloners are tech-savvy, at the very least and are usually well educated, with a focus on the sciences. The people \
 	of Ceti Episilon have developed a reputation for being some of the best and brightest that the Confederacy has to offer. However, everyone from Ceti Epsilon \
```
Comment (scrdest) [inline maps/nerva/fluff/locations_human.dm:119]: Can we fix these while we're here? 🥺 
```suggestion
	the cutting edge, most Epsiloners are tech-savvy, at the very least and are usually well educated, with a focus on the sciences. The people \
	of Ceti Epsilon have developed a reputation for being some of the best and brightest that the Confederacy has to offer. However, everyone from Ceti Epsilon \
```

File: maps/glloydstation/fluff/species_overrides.dm [hunk 2]
```diff
@@ -29,33 +29,52 @@
...
-			FACTION_POSITRONICS,
-			FACTION_SOL_CENTRAL,
+			FACTION_OTHER,
 			FACTION_NANOTRASEN,
-			FACTION_FREETRADE,
 			FACTION_XYNERGY,
-			FACTION_OTHER
+			FACTION_HEPHAESTUS,
+			FACTION_SOL_CENTRAL,
+			FACTION_INDIE_CONFED,
```
Comment (Glloyd) [inline maps/glloydstation/fluff/species_overrides.dm:70]: do we want IPCs from UHA?

Comment (Glloyd) [reply maps/glloydstation/fluff/species_overrides.dm:70]: I realize I wrote this in the Glloydstation file, even though this applies to Nerva, but this raises another question. Will this cause an issue, seeing as FACTION_INDIE_CONFED isn't available in glloydstation (see maps\glloydstation\fluff\_fluff.dm)?

Comment (ggermanicus) [reply maps/glloydstation/fluff/species_overrides.dm:70]: knew I forgot something. yeah faction uha is a bit too far and i copypasted too close to the sun in glloydstation overrides

**Comment by ErrorsWindows** [comment]
dang

**Pull request merged**